tests/Unit/NumericalAlgorithms/Spectral/Test_Mesh.cpp

```diff
@@ -242,6 +243,29 @@ void test_serialization() noexcept {
               {{Spectral::Quadrature::GaussLobatto, Spectral::Quadrature::Gauss,
                 Spectral::Quadrature::GaussLobatto}}});
 }
+
+void test_option_parsing() {
+  INFO("Option Parsing creation");
+
+  std::array<std::string, 3> bases{"Chebyshev", "Legendre", "FiniteDifference"};
+  std::array<std::string, 2> quadratures{"Gauss", "GaussLobatto"};
+
+  std::stringstream creation_string;
```

Comment:
Move this declaration inside the for loops, and then you don't need to clear it after each use.